import-export-cli/integration/apim/apim.go

```diff
@@ -2295,3 +2297,178 @@ func getTokenURL(host string, offset int) string {
 	port := 9443 + offset
 	return "https://" + host + ":" + strconv.Itoa(port) + "/oauth2/token"
 }
+
+// GetAPIPolicies : Get API Policies list of all types from APIM
+func (instance *Client) GetAPIPolicies(t *testing.T, offset, limit string) *APIPoliciesList {
+	var policyListResponse *APIPoliciesList
+
+	getPoliciesURL := instance.publisherRestURL + "/operation-policies"
+
+	queryParams := "offset=" + offset + "&limit=" + limit
+
+	if offset != "-1" && limit != "-1" {
+		getPoliciesURL += "?" + queryParams
+	}
+
+	request := base.CreateGet(getPoliciesURL)
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.GetAPIPolicies()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.GetAPIPolicies()", response, 200)
+
+	err := json.NewDecoder(response.Body).Decode(&policyListResponse)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return policyListResponse
+}
+
+// DeleteAPIPolicy : Deletes API Policy from APIM using UUID
+func (instance *Client) DeleteAPIPolicy(policyID string, baseTest string) {
+
+	policiesURL := instance.publisherRestURL + "/operation-policies/" + policyID
+
+	request := base.CreateDelete(policiesURL)
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.DeleteAPIPolicy()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	if baseTest == "Delete" {
+		base.ValidateAndLogResponse("apim.DeleteAPIPolicy()", response, 404)
+	} else if baseTest == "Emport" || baseTest == "Import" {
+		base.ValidateAndLogResponse("apim.DeleteAPIPolicy()", response, 200)
+	}
+}
+
+// GetAPIPolicy : Get API Policy from APIM using Id
+func (instance *Client) GetAPIPolicy(policyId string) map[string]interface{} {
+	var policyResponse map[string]interface{}
+
```

Comment:
Do we need all these new lines? After every line, we have used a new line for this function. Let's try to get them to meaningful blocks.

Reply:
Please check in other places as well

Reply:
@wasuradananjith WDYT? Do we have a standard that we need to follow here? New lines after every other code line seem redundant to me.

Reply:
There is no strict standards for this. Since other functions use the same formatting, let's keep the lines as they are.